modules/ROOT/pages/mule-error-concept.adoc

```diff
@@ -114,6 +114,16 @@ org.mule.runtime.core.internal.message.DefaultMessageBuilder$MessageImplementati
 }
 ----
 +
+Note that you can return the payload or attributes (metadata) 
+associated with the error message by using the following constructions:
+`#[error.errorMessage.payload]`, `#[error.errorMessage.attributes]`.
+Depending on the metadata supported by a given connector, you can
+also select values of fields from attributes, such as the status code from 
+and HTTP request (for example, `#[error.errorMessage.attributes.statusCode]`.
```

Comment:
from and HTTP request > from **an** HTTP request

Reply:
thanks, fixed